### PR TITLE
[WIP] New release 0.5.1, please add what's new 

### PR DESCRIPTION
**Pull request by robintibor**
So here this will become new version, last version 0.5 was 7 July 2020. Please consider to add Pull Requests of yours that have been merged since then @sliwy @gemeinl @hubertjb . Feel free to push directly to this PR. I will also add more from me. Probably easiest to use search function of Github to find stuff. 

Regarding sections, later we can also see if we want to come closer to https://keepachangelog.com/en/1.0.0/ 
Will leave this PR open for some time and ping again when I want to close it and create new release on PyPI.

### CODE REVIEW

**Comment by robintibor** [comment]
Just for reference previous brief discussion about what's new page was in this PR: https://github.com/braindecode/braindecode/pull/167

**Comment by robintibor** [comment]
You can search for your own PRs in our repo by searching like this:
```
is:pr author:robintibor 
```

File: docs/whats_new.rst [hunk 1]
```diff
@@ -27,7 +27,9 @@ Enhancements
 
 Bugs
 ~~~~
-- 
+- amplitude gradients are correctly computed for layers with multiple filters
+  (before, they were accidentally summed over all previous filters in the layer) (:gh:`167` by `Robin Tibor Schirrmeister`_)
+- get_output_shape and compute_amplitude_gradients assume 3d, not 4d inputs (:gh:`166` by `Robin Tibor Schirrmeister`_)
```
Comment (agramfort) [inline docs/whats_new.rst:32]: is get_output_shape and compute_amplitude_gradients part of the public API? if so it should be added to https://braindecode.org/api.html and then here you should be able to use :func:`...` to have a link in the what's new page.

**Comment by hubertjb** [comment]
@robintibor I tried to push my commit to your branch but got the following error:
```
 ! [remote rejected] robintibor/version-0-5-1 -> robintibor/version-0-5-1 (permission denied)
error: failed to push some refs to 'git@github.com:robintibor/braindecode-1.git'
```

Can you double check you've enabled edits on this PR?

**Comment by robintibor** [comment]
image
should be allowed? I thought?

**Comment by hubertjb** [comment]
> should be allowed? I thought?

It worked, thanks! I had to push to `HEAD:version-0-5-1` of your fork.

**Comment by agramfort** [comment]
@hubertjb you screwed up the diff :)

**Comment by hubertjb** [comment]
> @hubertjb you screwed up the diff :)

Did I do the push the wrong way?

**Comment by agramfort** [comment]
look at the diff I see things that I don't think should be there no?

>

**Comment by hubertjb** [comment]
Ah I see... I added some class imports in `__init__.py`s so that the objects could be referenced in `api.rst`. Should I move that to another PR instead?

**Comment by agramfort** [comment]
Ok great 👍 we should however not merge with green CIs

**Comment by robintibor** [comment]
Can you add a line for the now-merged PR with the parallel window stuff @hubertjb ?

**Comment by robintibor** [comment]
So we could make a new release now? Any objections? Otherwise I will just do it on monday, merging this and updating on PyPI @gemeinl @sliwy @hubertjb @agramfort

**Comment by agramfort** [comment]
no personal objections but I am not the most knowledgeable on current dev
efforts

>

**Comment by hubertjb** [comment]
works for me @robintibor !

**Comment by sliwy** [comment]
I added the line about fixing `CroppedTrialEpochScoring` in the `whats_new` file. Everything looks good from my side @robintibor !

**Comment by robintibor** [comment]
Although I think in 0.50 it wasn't broken? as this was before the changes to predict functions? like that's probably why it worked for our colleague before. but also fine to keep the note in :)

**Comment by sliwy** [comment]
@robintibor you're right, it was working before. Just remove my commit then!

**Comment by agramfort** [comment]
done https://twitter.com/agramfort/status/1404543292172275713?s=20

>

**Comment by robintibor** [comment]
amazing thanks :+1:

**Pull request merged**